lib/ecto/schema.ex

```diff
@@ -319,6 +320,16 @@ defmodule Ecto.Schema do
   end
 
   @doc """
+  TODO
+  """
+  defmacro schema(scope, source, [do: block]) do
```

Comment:
Great question. I was in doubt about this myself. I would like to talk to @chrismccord and @michalmuskala to be sure but, one of the points to have it not in an attribute, is to mirror the property that aliases usually have of being lexical. If we make it an attribute, people can put it inside `web/web.ex` and that's going to change how the code inside `schema` behaves in a non-obvious way.